websites/Q/Quran.com/presence.ts

```diff
@@ -0,0 +1,102 @@
+const presence = new Presence({
+		clientId: "969064871310282813",
+		injectOnComplete: true
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo_1024",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname } = window.location;
+	switch (true) {
```

Comment:
all you do in this switch statement is compare a variable to a string so please do a proper switch statement with the variable in here and the cases down below